dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImplIntegrationTest.java

```diff
@@ -0,0 +1,240 @@
+        assertFalse(drawedBodyJson.contains(host.getHostname()));
+        assertTrue(drawedBodyJson.contains(newHostname));
+
+        final Template workingVersion = (Template) APILocator.getVersionableAPI().findWorkingVersion(
+                template.getIdentifier(), APILocator.systemUser(), false);
+
+        assertFalse(workingVersion.getDrawedBody().contains(host.getHostname()));
+        assertTrue(workingVersion.getDrawedBody().contains(newHostname));
+
+        final FileAssetContainer containerFromDataBase = (FileAssetContainer) APILocator.getContainerAPI().getWorkingContainerByFolderPath(
+                FileAssetContainerUtil.getInstance().getFullPath(hostFromDataBse, ((FileAssetContainer) container).getPath()),
+                user, false,
+                null);
+
+        assertEquals(newHostname, containerFromDataBase.getHost().getName());
+    }
+
+    private Template createTemplate(final User user, final Host host, final Container container) {
+
+        final TemplateLayout templateLayout = TemplateLayoutDataGen.get()
+                .withContainer(container)
+                .next();
+
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        return new TemplateDataGen()
+                .drawedBody(templateLayout)
+                .body(String.format(body, host.getHostname()))
+                .host(host)
+                .theme(theme)
+                .nextPersisted();
+    }
+
+    private Container createContainer(final User user, final Host host) throws DotSecurityException, DotDataException {
+        final ContentType contentType = new ContentTypeDataGen()
+                .host(host)
+                .nextPersisted();
+
+        Container container = new ContainerAsFileDataGen()
+                .contentType(contentType, "")
+                .host(host)
+                .nextPersisted();
+
+        container = APILocator.getContainerAPI().find(container.getInode(), user, false);
+        return container;
+    }
+
+    private void waitUntilJobIsFinish() throws InterruptedException, SchedulerException {
+        Thread.sleep(500);
+
+        while(true){
+            final String[] jobGroupNames = QuartzUtils.getSequentialScheduler().getJobGroupNames();
+
+            if (jobGroupNames.length > 0 &&
+                    Arrays.asList(jobGroupNames).contains("update_containers_paths_job") &&
+                    !QuartzUtils.getSequentialScheduler().getCurrentlyExecutingJobs().isEmpty()){
+                Thread.sleep(500);
+            } else {
+                break;
+            }
+        }
+    }
+
+    /**
+     * Method to Test: {@link ContentletWebAPIImpl#saveContent(Map, boolean, boolean, User)}
+     * When: Not Change a Host' name and exists any template using this
+     * Should: Not change anything into the template layout
+     *
+     * */
+    @Test
+    public void whenHostNameNotChange() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5990100)